Katello/katello
# Fixes #29753 - Support skipping srpms

**To test:**
1. Create yum repo with srpm content remote. ex: (https://repos.fedorapeople.org/repos/pulp/pulp/fixtures/srpm/)
2. Chose srpm as ignorable_content when creating repo.
3. Sync and verify that srpm content is not synced.
